Gem/Code/Source/Weapons/WeaponTypes.cpp

```diff
@@ -268,22 +298,26 @@ namespace MultiplayerSample
         }
     }
 
+    constexpr AZ::Uuid s_popcornFxAssetUuid{ "{45047C35-64F7-43BA-B463-000081B587C3}" };
```

Comment:
Please switch over to using the exposed AssetId now that we've got one from the PopcornFX gem.